app/controllers/annotations_controller.rb

```diff
@@ -18,7 +18,8 @@ def admin_create
     guid_save = guidance.present? ? guidance.save : true
 
     if ex_save && guid_save
-      redirect_to admin_show_phase_path(id: @question.section.phase_id, section_id: @question.section_id, question_id: @question.id, edit: 'true'), notice: _('Information was successfully created.')
+      typ = (!ex_save && !guid_save ? 'example answer and guidance' : (!guid_save ? 'guidance' : 'example answer'))
```

Comment:
because of the above if statement will requires both ex_save and guid_save to be true, this code will always return 'example answer'.  This should be revised with `example_answer.present?` and `guidance.present?`